extra/intel-tbb/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# $Id$
+# Maintainer: Stéphane Gaudreault <stephane@archlinux.org>
+# Contributor: Thomas Dziedzic < gostrc at gmail >
+# Contributor: Denis Martinez <deuns.martinez AT gmail.com>
+
+# ALARM: Romain Reignier <rom.reignier AT gmail.com>
+# - add armv7h architecture
+# - add CXXFLAGS for armv7h
+
+buildarch=4
+
+pkgname=intel-tbb
+pkgver=4.3_20150209
+pkgrel=1.1
+pkgdesc='High level abstract threading library'
+arch=('i686' 'x86_64' 'armv7h')
```

Comment:
Listing armv7h isn't technically necessary, our build system ignores the arch field.